superset/assets/javascripts/explorev2/stores/store.js

```diff
@@ -1718,3 +1718,9 @@ export function initialState(vizType = 'table') {
     isStarred: false,
   };
 }
+
+// Control Panel fields that re-render chart without need for 'Query button'
+export const autoQueryFields = [
+  'datasource',
```

Comment:
i was curious why we decided on just these two fields for auto-querying, but this is what we do in v1 currently, so replicating in v2. @vera-liu and i had some discussion about whether all fields should be auto-queryable. there are other such as `bottom-margin`, `show-legend`, `rich-tooltip` but we will tackle those in another PR.